README.md

```diff
@@ -128,6 +128,7 @@ On a CI system that runs on submitted PRs, we determine what commits to include
 
 But what if we want to set up a continuous deployment system
 that, as changes get pushed to `master`, builds and deploys
+only the affected projects?
```

Comment:
What's up with the newlines within this paragraph? :)

Reply:
That got lost on another commit. I have fixed it on main and rebased to this PR